src/main/scala/com/fulcrumgenomics/wgs/CreateWgsCallableIntervalList.scala

```diff
@@ -0,0 +1,126 @@
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.wgs
+
+import com.fulcrumgenomics.cmdline.{FgBioTool, ClpGroups}
+import com.fulcrumgenomics.util.ProgressLogger
+import dagr.commons.CommonsDef._
+import dagr.commons.io.Io
+import dagr.commons.util.LazyLogging
+import dagr.sopt._
+import dagr.sopt.cmdline.ValidationException
+import htsjdk.samtools.util.SamLocusIterator.LocusInfo
+import htsjdk.samtools.util.{IntervalList, Interval, SamLocusIterator, CloserUtil}
+import htsjdk.samtools.{SAMFileHeader, SamReaderFactory, SamReader}
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable.ListBuffer
+
+/**
+  * Creates a list of intervals for WGS variant calling.
+  */
+@clp(description =
+  """
+    |Creates a list of intervals for WGS variant calling.
+    |
+    |The output intervals will only contain sites that have both a minimum and maximum coverage as set by the user.
+    |This tool is useful for when sites with too much coverage cause too much slow down in variant calling.
+    |
+    |When accumulating records per locus, we scale the maximum number to be accumulated to reduce any bias during
+    |the accumulation process (see `SamLocusIterator.setMaxReadsToAccumulatePerLocus`).
+  """,
+  group = ClpGroups.Utilities)
+class CreateWgsCallableIntervalList
+( @arg(flag="i", doc="Input BAM file.") val input: PathToBam,
+  @arg(flag="o", doc="Output interval list") val output: PathToIntervals,
+  @arg(flag="m", doc="Minimum coverage.") val minCoverage: Int = 1,
+  @arg(flag="M", doc="Maximum coverage.") val maxCoverage: Int = 1024,
+  @arg(flag="f", doc="Scale the maximum number of reads to accumulate at any position relative to the maximum.") val maxToAccumulatePerLocus: Float = 2.0f
+) extends FgBioTool with LazyLogging {
+
+  Io.assertReadable(input)
+  Io.assertCanWriteFile(output)
+  if (maxToAccumulatePerLocus < 1.0f) throw new ValidationException("The scaling factor for maximum number of reads to accumulate per locus must be => 1.")
+
+  override def execute(): Unit = {
+    val progress: ProgressLogger = new ProgressLogger(logger, noun="positions", unit=10*1000*1000)
+    val in: SamReader = SamReaderFactory.make.open(input.toFile)
+    val locusIterator = new SamLocusIterator(in)
```

Comment:
I think you probably want to `setIncludeIndels()` here too, otherwise you'll exclude any sites that are homozygously deleted.